contents/blog/best-open-source-analytics-tools.md

```diff
@@ -66,21 +73,24 @@ PostHog is useful for engineering, data science, and product teams. As an all-in
 - Event pipelines to integrate with data warehouses
 - Built on the ultra-fast ClickHouse OLAP database
 
-#### Open source license and monetization
+#### License and monetization
 
-[PostHog Open Source](https://github.com/PostHog/posthog) is free to use for life and is distributed under an MIT license. The open source version includes the core product analytics, feature flag, and session recording features, but has a one project limit. There's also a [FOSS edition](https://github.com/PostHog/posthog-foss) that removes all proprietary code. 
+[PostHog Open Source](https://github.com/PostHog/posthog) is free to use for life and distributed under an MIT license. The open-source version includes the core product analytics, feature flag, and session recording features, but has a one project limit. There's also a [FOSS edition](https://github.com/PostHog/posthog-foss) that removes all proprietary code. 
 
 Subscribing to PostHog Cloud removes the project limit and adds numerous paid-only features, including experimentation, correlation analysis, group analytics for tracking organizations, and advanced cohorts. PostHog Cloud is [free up to 1 million events per month](/pricing).
 
```

Comment:
CTA? 😅

Reply:
I want to keep this free of marketing CTAs etc, tbh.